apps/strapi-cms/Dockerfile.prod

```
@@ -0,0 +1,29 @@
+# Creating multi-stage build for production
+FROM node:18-alpine as build
+RUN apk update && apk add --no-cache build-base gcc autoconf automake zlib-dev libpng-dev vips-dev > /dev/null 2>&1
+ARG NODE_ENV=production
+ENV NODE_ENV=${NODE_ENV}
+
+WORKDIR /opt/
+COPY package.json package-lock.json ./
+RUN npm config set fetch-retry-maxtimeout 600000 -g && npm install --only=production
```

```suggestion
RUN npm config set fetch-retry-maxtimeout 600000 -g && npm install
```
if you set the `NODE_ENV` to production, this flag is not necessary